proposals/0158-cloud-app-transport-adapter.md

```diff
@@ -115,7 +115,9 @@ Add new RPCs `SetCloudAppProperties` and `GetCloudAppProperties`.
     </enum>
 
     <struct name="CloudAppProperties" since="5.x">
-        <param name="appName" type="String" maxlength="100" mandatory="true"></param>
+        <param name="appName" type="String" minlength="0" maxlength="100" array="true" minsize="0" maxsize="100" mandatory="false">
```

Comment:
Should this stay as `appName`? Or should we change it to `appNames` since it is an array? Is it already baked into the policy table stuff?

Reply:
This value is named `nicknames` in the policy table, so since the purpose of this parameter has changes, we could just copy the policy table naming scheme and rename this parameter to `nicknames` as well.